client/state/initial-state.js

```diff
@@ -16,6 +17,8 @@ import config from 'config';
  */
 const debug = debugModule( 'calypso:state' );
 
+export const MAX_AGE = ms( '7 days' );
```

Comment:
It's nice on the PHP side of things because we use constants that are easily searchable

```js
const MAX_AGE = 7 * DAY_IN_HOURS * HOUR_IN_MS
```

or something like that